### PR TITLE
Fix UnboundLocalError in corpus pruning task.

### PR DESCRIPTION
**Pull request by mpherman2**
Still looking though logs to confirm that this is the bug, but this looks wrong to me. PTAL.

### CODE REVIEW

File: src/python/bot/tasks/corpus_pruning_task.py [hunk 1]
```diff
@@ -513,8 +513,8 @@ def run(self, timeout):
           'Shared corpus merge finished successfully.',
           output=symbolized_output)
     except engine.TimeoutError as e:
-      logs.log_error('Corpus pruning timed out while merging shared corpus: ' +
-                     repr(e))
+      raise CorpusPruningException(
+          'Corpus pruning timed out while merging shared corpus: ' + repr(e))
```
Comment (Dor1s) [inline src/python/bot/tasks/corpus_pruning_task.py]: use `\n` at the end of the line, otherwise all these errors will be reported as unique

**Comment by inferno-chromium** [comment]
No this is wrong, we explicitly didn't want to raise an exception for this case as it happens often.

**Comment by mbarbella-chromium** [comment]
Ah, makes sense. I'll revert this.

**Pull request merged**